releasenotes/notes/dogstatsd-mapper-668ce195a8c045b5.yaml

```
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Add mapping feature to dogstatsd to convert parts of dogstatsd/statsd
```

what do you think of keeping just `statsd` here?
```suggestion
    Add mapping feature to dogstatsd to convert parts of statsd
```

---

Technically, it works for `dogstatsd` packets too, so just indicating `statsd` might be misleading.